src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  &--4 :nth-child(-n + 4) {
```

Comment:
There is an extra space between `&--4` and `:nth-child(-n + 4)`. According to the code style rules, there should be no space between the element and the pseudo-class.